modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -0,0 +1,270 @@
+= Gateway Startup Encryption in Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information stored by the runtime such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.
+
+== Encrypting Your Platform Credentials Using the Agent
+
+Configuring encryption through the Runtime Manager Agent is the fastest way to secure your client credentials. You have to add the encryption key when creating an Agent server, and the agent will handle the properties encryption for you. +
+See xref:runtime-manager::installing-and-configuring-runtime-manager-agent.adoc[Install and Configure the Runtime Manager Agent] to learn how to configure the agent with encryption.
+
+== Encrypting the Client ID and Secret
+
+If you are not using an agent, you can still manually encrypt your properties.
+
+The most common way, is using the xref:mule-runtime::secure-configuration-properties.adoc#secure-properties-tool[Secure Properties Tools]. +
+When using encrypted properties, the API gateway assumes that you are using the AES algorithm in CBC Mode.
+
+=== Example
+
+Suppose that your client ID is `bb458e566cba4e2ea5a826d27b46031a`, and that you choose the encryption key to be the 128 bits string `MyEncryptionKey1`. To encrypt it, you need to run the following command:
+
+// tag::example[]
+[source,console,linenums]
+--
+> java -jar secure-properties-tool.jar \
+string \ //<1>
+encrypt \ //<2>
+AES \ //<3>
+CBC \
+MyEncryptionKey1 \ //<4>
+bb458e566cba4e2ea5a826d27b46031a
+--
+
+<1> As you are trying to encrypt a single value, you need to specify it using the string parameter.
+<2> Set the tool to encrypt the value.
+<3> Use AES with CBC Mode. +
+This is important, as the runtime only uses this configuration to decrypt the value later.
+<4> And finally, you write your encryption key and the value you want to encrypt.
+// end::example[]
+
+This command outputs your encrypted client ID:
+
+[source]
+--
+> smzbg+aV2HbiC9Fv8KBsZpD5PlXiplsfhZ42RWNEbHhUeZ9ZIDftljUx6sYpuQ+b
+--
+
+== Passing your Encrypted Client ID and Secret
+
+After encrypting your credentials, you need to configure your encrypted values in your `wrapper.conf` file located inside the `conf` directory in the root folder of your Mule runtime:
+
+[source]
+--
+anypoint.platform.client_id.<n>
+anypoint.platform.client_secet.<n>
```

Comment:
Seems this is a typo: anypoint.platform.client_secet.<n>, should be anypoint.platform.client_secret.<n>